docs/en/api/FS-API.md

```diff
@@ -187,23 +212,40 @@ Users can override the default policy class in the
 [configuration file]({{ '/en/operation/Configuration.html' | relativize_url }}) at property
 `alluxio.user.block.write.location.policy.class`. The built-in policies include:
 
-* **LocalFirstPolicy (alluxio.client.block.policy.LocalFirstPolicy)**
+* [LocalFirstPolicy](https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/alluxio/client/block/policy/LocalFirstPolicy.html)
+
+  **This is the default policy.**
+
+  > A policy that returns local host first, and if the local worker doesn't have enough availability,
+  > it randomly picks a worker from the active workers list for each block write.
+
+* [LocalFirstAvoidEvictionPolicy](https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/alluxio/client/block/policy/LocalFirstAvoidEvictionPolicy.html)
+
+  This is the same as `LocalFirstPolicy` with the following addition:
 
-    Returns the local worker first, and if it does not have enough capacity of a block,
-    randomly picks a worker from the active workers list. This is the default policy.
+  > If no worker meets the demands, return local host. USER_FILE_WRITE_AVOID_EVICTION_POLICY_RESERVED_BYTES
+  > is used to reserve some space of the worker to store the block, for the values
+  > mCapacityBytes minus mUsedBytes is not the available bytes.
 
-* **MostAvailableFirstPolicy (alluxio.client.block.policy.MostAvailableFirstPolicy)**
+* [MostAvailableFirstPolicy](https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/alluxio/client/block/policy/MostAvailableFirstPolicy.html)
 
-    Returns the worker with the most available bytes.
+  > A policy that returns the worker with the most available bytes. The policy returns null if no worker is qualified.
 
-* **RoundRobinPolicy (alluxio.client.block.policy.RoundRobinPolicy)**
+* [RoundRobinPolicy](https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/alluxio/client/block/policy/RoundRobinPolicy.html)
 
-    Chooses the worker for the next block in a round-robin manner and skips workers that do not have
-    enough capacity.
+  > A policy that chooses the worker for the next block in a round-robin manner
+  > and skips workers that do not have enough space. The policy returns null if no worker can be found.
 
-* **SpecificHostPolicy (alluxio.client.block.policy.SpecificHostPolicy)**
+* [SpecificHostPolicy](https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/alluxio/client/block/policy/SpecificHostPolicy.html)
 
-    Returns a worker with the specified host name. This policy cannot be set as default policy.
+  > Always returns a worker with the specified hostname. Returns null if no active worker on that hostname found.
```

Comment:
did you purposefully omit the `This policy cannot be set as the default policy` line?

Reply:
Since I copied the description verbatim from the Javadoc, I guess so yeah. I didn't notice that so I'll add it back outside of the quoted block, thanks!